components/header.js

```diff
@@ -31,3 +32,19 @@ const StyledHeadline = styled.h1`
   flex-grow: 1;
   text-align: center;
 `;
+
+const HeaderBox = styled.header`
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 3.8rem;
+  background-color: #fdfefe;
+  width: 100vw;
+  color: black;
```

Comment:
```suggestion
  color: blue;
```
would say looks better